book/references.bib

```diff
@@ -36,3 +34,70 @@ @misc{2022_IS2_HW_tutorials
+	language = {en},
+	number = {7},
+	urldate = {2021-01-04},
+	journal = {PLOS ONE},
+	author = {Huppenkothen, Daniela and McFee, Brian and Norén, Laura},
+	month = jul,
+	year = {2020},
+	note = {Publisher: Public Library of Science},
+	keywords = {Algorithms, Careers, Culture, Decision making, Employment, Human learning, Statistical distributions, Workshops},
+	pages = {e0231939},
+	file = {Snapshot:C\:\\Users\\arend\\Zotero\\storage\\CYPGLSAA\\article.html:text/html},
+}
+
+@article{Leong2021, 
+author = "Wei Ji Leong", 
+title = "{The subglacial landscape and hydrology of Antarctica mapped from space}", 
+year = "2021", 
+month = "7", 
+url = "https://openaccess.wgtn.ac.nz/articles/thesis/The_subglacial_landscape_and_hydrology_of_Antarctica_mapped_from_space/14956062", 
+doi = "10.26686/wgtn.14956062.v1" 
+} 
+
+@article{siegfried2014,
+author = {Siegfried, Matthew R. and Fricker, Helen A. and Roberts, Mackenzie and Scambos, Ted A. and Tulaczyk, Slawek},
+title = {A decade of West Antarctic subglacial lake interactions from combined ICESat and CryoSat-2 altimetry},
+journal = {Geophysical Research Letters},
+volume = {41},
+number = {3},
+pages = {891-898},
+keywords = {subglacial lakes, altimetry, CryoSat-2, ICESat, glacier dynamics, ice stream},
+doi = {https://doi.org/10.1002/2013GL058616},
+url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1002/2013GL058616},
+eprint = {https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1002/2013GL058616},
+abstract = {We use CryoSat-2 interferometric satellite radar altimetry over the Mercer and Whillans ice streams, West Antarctica, to derive surface elevation changes due to subglacial lake activity at monthly resolution for the period 2010 to 2013. We validate CryoSat-2 elevation measurements, trends, and spatial patterns of change using satellite image differencing and in situ vertical movement from Global Positioning System (GPS) data. Two subglacial lake discharge events occur in the same subglacial-hydrological catchment within a 9 month period. Using GPS measurements that are spanning the gap between the Ice, Cloud, and land Elevation Satellite and Cryosat-2 missions, we cross-calibrate the two missions to establish the efficacy of CryoSat-2 altimetry to measure dynamic changes on the ice sheets.},
+year = {2014}
+}
```

Comment:
Both of these references don't seem to show up correctly:

![image](https://user-images.githubusercontent.com/23487320/220767199-dbcc9312-b5c2-4d04-b41d-640e8337aaad.png)

The thesis isn't showing up, and I'm not sure why the Siegfried et al. paper is showing up like an arXiV citation :sweat_smile:

Reply:
I addressed this issue in this [PR](https://github.com/geo-smart/deepicedrainpipe/pull/16).